File: app/src/androidTest/java/com/example/bernatvarela/espressotestapp/presentation/pages/MainPage.java
package com.example.bernatvarela.espressotestapp.presentation.pages;

import com.example.bernatvarela.espressotestapp.R;

public class MainPage {
    private static final String movie = "Movie";
    private static final int list = R.id.mainListView;
    private String stringToBeFound = "Movie: 66";
    private String partStringToBeFound = "Movie: 70";

    public MainPage() {}

    public int getList() {
        return list;
    }

    public String getPartStringToBeFound() {
        return partStringToBeFound;
    }

    public String getMovie() {
        return movie;
    }

    public String getStringToBeFound() {
        return stringToBeFound;
    }
}

File: app/src/androidTest/java/com/example/bernatvarela/espressotestapp/presentation/pages/MainPage.kt
package com.example.bernatvarela.espressotestapp.presentation.pages

import com.example.bernatvarela.espressotestapp.R

class MainPage {
    val stringToBeFound = "Movie: 66"
    val partStringToBeFound = "Movie: 70"

    fun getList(): Int {
        return list
    }

    fun getMovie(): String {
        return movie
    }

    companion object {
        private val movie = "Movie"
        private val list = R.id.mainListView
    }
}
